### PR TITLE
pubsub java: make deprecated methods package-private

### PR DESCRIPTION
**Pull request by pongad**
updates GoogleCloudPlatform/google-cloud-java#1828.

### CODE REVIEW

File: google/pubsub/v1/pubsub_gapic.yaml [hunk 1]
```diff
@@ -123,6 +123,11 @@ interfaces:
     retry_codes_name: idempotent
     retry_params_name: default
     timeout_millis: 60000
+    surface_treatments:
+    - include_languages:
+      - java
+      release_level: DEPRECATED
```
Comment (garrettjonesgoogle) [inline google/pubsub/v1/pubsub_gapic.yaml]: Remove `DEPRECATED` from all of the package-private methods now.

Comment (pongad) [inline google/pubsub/v1/pubsub_gapic.yaml]: Done.

**Comment by pongad** [comment]
@garrettjonesgoogle PTAL

**Comment by garrettjonesgoogle** [comment]
LGTM

**Pull request merged**